lib/puppet/module.rb

```diff
@@ -68,6 +68,17 @@ def initialize(name, path, environment)
     load_metadata
 
     @absolute_path_to_manifests = Puppet::FileSystem::PathPattern.absolute(manifests)
+
+    # i18n initialization for modules
+    if Puppet::GETTEXT_AVAILABLE
```

Comment:
Loading gettext happens very early, and I seem to remember there being a reason we couldn't use Puppet features because of that. If that's not the case, this CAN probably change; the constant is only used internally.